static/app/components/globalDrawer/components.tsx

```diff
@@ -0,0 +1,96 @@
+import {useEffect, useRef, useState} from 'react';
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import type {DrawerOptions} from 'sentry/components/globalDrawer';
+import {IconClose} from 'sentry/icons/iconClose';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import useKeyPress from 'sentry/utils/useKeyPress';
+import useOnClickOutside from 'sentry/utils/useOnClickOutside';
+import SlideOverPanel from 'sentry/views/insights/common/components/slideOverPanel';
```

Comment:
We should pull this out into the top level components directory (and perhaps into this folder?) if it's a subcomponent here.